state_machine/drivers/emulation/lib/pycubed.py

```diff
@@ -43,26 +45,34 @@ def __init__(self):
         self.c_state_err = 0
         self.c_boot = None
 
+        schedule = tasko.schedule
+        rt = reader_task(self)
+        self.reader_task = schedule(5, rt.main_task, 10)  # try to read from stdin at 5Hz
+
+        self._accel = (0.0, 0.0, 0.0)
+        self._mag = (0.0, 0.0, 0.0)
+        self._gyro = (0.0, 0.0, 0.0)
+
     def new_file(self, substring, binary=False):
         print(
-            f"new file not implemented, {substring} not written with binary={binary}")
+            f"new file not implemented, string not written with binary={binary}")
         return None
 
     @property
     def acceleration(self):
-        return (0.0, 0.0, 0.0)
+        return self._accel
 
     @property
     def magnetic(self):
-        return (0.8, 0.3, -0.3)
+        return self._mag
```

Comment:
Out of curiosity, why are we changing the spoofed sensor readings to all 0s?

Reply:
I just set them to be initially zero, but you can now put things into stdin to change the sensor reading values. Honestly this is far from being ready for code review.